### PR TITLE
Add more async support

### PR DESCRIPTION
**Pull request by jettify**
PR for #92 #91

### CODE REVIEW

**Comment by jettify** [comment]
Fixed issue with possible double commit. Please review.

File: aiopg/sa/result.py [hunk 1]
```diff
@@ -5,9 +5,18 @@
 
 from sqlalchemy.sql import expression, sqltypes
 
+from ..utils import PY_35
 from . import exc
 
 
+try:
+    StopAsyncIteration
```
Comment (asvetlov) [inline aiopg/sa/result.py]: Why do you need the check?

`if PY_35:` statement above prevents `StopAsyncIteration` usage for python < 3.5

**Comment by asvetlov** [comment]
Looks good but probably we should get rid of `StopAsyncIteration` in `utils.py` too.

**Comment by jettify** [comment]
I have dropped  `StopAsyncIteration` but pyflakes complains with:
```python
aiopg/cursor.py:392:23: F821 undefined name 'StopAsyncIteration'
aiopg/sa/result.py:337:23: F821 undefined name 'StopAsyncIteration'
```
so I have added `# noqa` marker to those lines.

**Comment by asvetlov** [comment]
LGTM

**Pull request merged**